src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateTargetCoverage.java

```diff
@@ -57,8 +57,9 @@
  * </pre>
  *
  * <p>
- *     Panel of Normal (PoN) creation uses the --transform PCOV parameter as shown.
- *     To collect raw coverage for tumor sample copy number analysis, omit this option.
+ *     To collect raw coverage, omit the --transform PCOV option.
```

Comment:
Perhaps something like: `To collect coverage in terms of integer counts (i.e., not normalized by the total number of targets), omit the --transform PCOV option.

Note: The use of "raw" in subsequent documentation may refer to either (i) coverage in terms of integer counts from this step or (ii) non-denoised coverage in general, which may be in terms of either integer counts or proportional coverage.`

And do we really use (ii) in the javadoc, or does that just happen in the tutorial? If the latter, then I would omit the note entirely. But in any case, I think you want to use the word "integer" somewhere.